scripts/conflict_resolver.py

```
@@ -0,0 +1,312 @@
+                    "rule_name": c.rule_name,
+                    "description": c.description,
+                    "directory": c.directory,
+                    "severity": c.severity,
+                    "found_files": c.found_files,
+                    "preferred_file": c.preferred_file,
+                    "conflicting_files": c.conflicting_files
+                }
+                for c in conflicts
+            ],
+            "removed_files": self.removed_files,
+            "failed_removals": self.failed_removals
```

Ruff (0.11.9)

Dictionary key literal `"failed_removals"` repeated

(F601)